lib/beaker/dsl/helpers/web_helpers.rb

```diff
@@ -47,7 +47,7 @@ def fetch_http_file(base_url, file_name, dst_dir)
           base_url.chomp!('/')
           src = "#{base_url}/#{file_name}"
           dst = File.join(dst_dir, file_name)
-          if File.exists?(dst)
+          if options[:cache_files_locally] && File.exists?(dst)
```

Comment:
Should a warning to `STDOUT` be put here to provided notice of the up-coming change to change `:cache_files_locally` from `true` to `false`? Or will that change be communicated later?